File: WebContent/js/registration.js
function checkForValidation()
{
	var errorMessage=null;
	var username=document.registration.username.value;
	var password=document.registration.password.value;
	var name=document.registration.name.value;
	var gender=document.registration.gender.value;
	var city=document.registration.city.value;
	if(username=="" || username==null)
		{
			errorMessage +="please Enter User Name \n";
		}
	if(password=="" || password==null)
		{
		errorMessage +="please Enter password \n";
		}
	if(name=="" || name==null)
		{
		errorMessage +="please Enter name \n";
		}
	if(gender=="" || gender==null)
		{
		errorMessage +="please Select gender \n";
		}
	if(city=="" || city==null)
		{
		errorMessage +="please Select city \n";
		}
	if(errorMessage!=null)
		{
		alert(errorMessage);
		return false;
		}
	}

File: src/com/controller/Registration.java
package com.controller;

import java.io.IOException;

import javax.servlet.RequestDispatcher;
import javax.servlet.ServletException;
import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

import com.data.DAO;
import com.model.User;

/**
 * Servlet implementation class Registration
 */
@WebServlet("/Registration")
public class Registration extends HttpServlet {
	private static final long serialVersionUID = 1L;

	/**
	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
	 */
	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
		String uid=request.getParameter("username");
		String pass=request.getParameter("password");
		String name=request.getParameter("name");
		String gen=request.getParameter("gender");
		String city=request.getParameter("city");
		String message=null;
		String resource="registration.jsp";
		if(uid==null || uid.trim().equals("")){
			message="Please enter UserID";
		}
		else if(pass==null || pass.trim().equals("")){
			message="Please enter Password";
		}
		else if(name==null|| name.trim().equals("")){
			message="Please enter name";
		}
		else if(gen==null||gen.equals("")){
			message="Please enter gender";		
			
		}
		else if(city==null||city.equals("")){
			
		}
		else{
			User usr=new User();
			usr.setUserid(uid);
			usr.setPassword(pass);
			usr.setName(name);
			usr.setGender(gen);
			usr.setCity(city);
			try {
				DAO.saveUser(usr);
				message="Registered Successfully :Please Login";
				resource="login.jsp";
			} catch (Exception e) {
				message=e.getMessage();
			}
			
		}
		request.setAttribute("msg", message);
		RequestDispatcher rd=request.getRequestDispatcher(resource);
		rd.forward(request, response);
	}

}
